### ISSUE TITLE
make fail with llvm4.0

### ISSUE THREAD

**Issue by suulnnka** [dibyendumajumdar/ravi#124]
fail in include/ravi_llvm.h:30
```
#if (LLVM_VERSION_MAJOR != 3 || LLVM_VERSION_MINOR < 5)
#error Unsupported LLVM version
#endif
```

**Comment by dibyendumajumdar** [comment]
Thank you for the report. I am waiting for the final 4.0 release before testing. Did you find any other issues other than the one above?

**Comment by dibyendumajumdar** [comment]
Resolved in 0.20
